pyrogram/client.py

```diff
@@ -331,16 +331,29 @@ async def updates_watchdog(self):
                 await self.invoke(raw.functions.updates.GetState())
 
     async def listen(
-        self,
-        filters: Optional[Filter] = None,
-        listener_type: ListenerTypes = ListenerTypes.MESSAGE,
-        timeout: Optional[int] = None,
-        unallowed_click_alert: bool = True,
-        chat_id: int = None,
-        user_id: int = None,
-        message_id: int = None,
-        inline_message_id: str = None,
+            self,
```

Comment:
Why the extra spaces here and in other places? This isn't a problem, just a nitpick, as it generate a bigger diff.
In vscode (and probably other editors) you can easily decrease the indentation level by using <kbd>ctrl</kbd> + <kbd>[</kbd>